uxarray/grid/grid.py

```diff
@@ -1064,6 +1214,112 @@ def compute_face_areas(
 
         return self._face_areas, self._face_jacobian
 
+    def to_netcdf(
```

Comment:
as per discussion with Orhan, update `Grid.encode_as` to be chained with Xarray functionality (i.e. `to_netcdf` or `to_zarr`)

Reply:
Also, possible name changes to `encode_as`